storage/buckets.rb

```diff
@@ -90,6 +90,86 @@ def create_bucket project_id:, bucket_name:
   # [END create_bucket]
 end
 
+def create_bucket_class_location project_id:, bucket_name:, location:, storage_class:
+  # [START create_bucket_class_location]
+  # project_id    = "Your Google Cloud project ID"
+  # bucket_name   = "Name of Google Cloud Storage bucket to create"
+  # location      = "Location of your Google Cloud Storage bucket"
+  # storage_class = "Storage class of your Google Cloud Storage bucket"
```

Comment:
What is an example of what this looks like?

I wonder if it would be more clear if `location` and `storage_class` were explicit rather than variables.

For example of explicit options, see Speech Recognize:

```ruby
audio = speech.audio audio_file_path, encoding:    :linear16,   # <--- All Hard Coded Values
                                      sample_rate: 16000,       #
                                      language:    "en-US"      #
```

I know that this sample will appear within the documentation which should mention the available values, but I can't personally even *guess* what a correct value for `storage_class`might be

Reply:
Understood, will update to use explicit values instead. I got used to variable, and the canonical uses explicit values.